### PR TITLE
Overall README changes for up-to-date

### PR DESCRIPTION
**Pull request by fpscan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +33,12 @@ Latest binaries are currently hosted on the [buildbot](http://buildbot.libretro.
 
 ## Support
 
-To reach developers, either make an issue here on GitHub, make a thread on the [forum](http://www.libretro.com/forums/), chat on [discord](https://discord.gg/C4amCeV), or visit our IRC channel: #retroarch @ irc.freenode.org.
+To reach developers, either make an issue here on GitHub, make a thread on the [forum](http://www.libretro.com/forums/), chat on [discord](https://discord.gg/C4amCeV), or visit our IRC channel: #retroarch @ irc.freenode.org. You could create a post in [Reddit](https://www.reddit.com/r/RetroArch/) with *Technical Support* flair.
 
 ## Documentation
 
 See our [Documentation Center](https://docs.libretro.com/). On Unix, man-pages are provided.
-More developer-centric stuff is found [here](https://github.com/libretro/libretro.github.com/wiki/Documentation-devs).
```
Comment (fpscan) [inline README.md]: since it's no longer active.

File: README.md [hunk 2]
```diff
@@ -8,11 +8,13 @@ Popular examples of implementations for this API includes video game system emul
 more generalized 3D programs.
 These programs are instantiated as dynamic libraries. We refer to these as "libretro cores".
 
-![XMB menu driver](http://i.imgur.com/BMR1xxr.png "XMB menu driver")
+![XMB menu driver](https://github.com/libretro/retroarch-scene/blob/master/Scratchpad/images/Versions/1.8.4/XMB-main-menu.jpg?raw=true "XMB menu driver")
```
Comment (RobLoach) [inline README.md]: https://github.com/libretro/retroarch-scene ? We could likely upload the screenshots directly to this repo to reference the images by reference.

For example....

```
![XMB Menu Driver](docs/screenshot1.png)
```

Comment (fpscan) [reply README.md]: hmm, does it work like that?

Comment (RobLoach) [reply README.md]: You can see a demo of that over at https://github.com/RobLoach/chailove-floppybird ....

```
![Floppy Bird Gameplay](screenshot.png)
[![Floppy Bird Gameplay](screenshot.png)](https://www.youtube.com/watch?v=RLVwTh6qDFI&hd=1)
```

Comment (fpscan) [reply README.md]: Okay I totally missunderstand, I thought we talking docs repo itself instead docs folder in RetroArch.

So it should be okay now https://github.com/libretro/RetroArch/blob/b44968eea57807f623227827a817a868f48b0942/README.md

File: README.md [hunk 3]
```diff
@@ -256,3 +259,23 @@ These modelines are more accurate giving exact hz. However, some games may have
...
+- YouTube: https://www.youtube.com/Libretro
+- Google Post: https://posts.google.com/share/55Nhs2jG
+- Steam: https://store.steampowered.com/app/1118310/RetroArch/
+- YouTube Topic: https://www.youtube.com/channel/UC5q007PYyQPgin0HHbzF0zQ
+- Patreon: https://www.patreon.com/libretro
+- BOUNTYSOURCE: https://www.bountysource.com/teams/libretro/issues
+- Discord: https://discord.gg/27Xxm2h
+- Teespring: https://teespring.com/stores/retroarch
+- Documentation: http://docs.libretro.com/
+- Forums: https://forums.libretro.com/
```
Comment (RobLoach) [inline README.md]: Could probably use Markdown links here. For example:
```
- [YouTube](https://www.youtube.com/Libretro)
```

Comment (fpscan) [reply README.md]: I wasn't sure if it look good so i didn't make it as Markdown in first place.

File: README.md [hunk 4]
```diff
@@ -256,3 +259,23 @@ These modelines are more accurate giving exact hz. However, some games may have
 Some arcade resolutions can be very different from consumer CRTs. There is resolution detection to ensure MAME games will be displayed in the closest available resolution but drawn at their native resolution within this resolution. Meaning that the MAME game will look just like the original hardware.
 
 MAME ROMs that run in a vertical aspect like DoDonPachi need to be rotated within MAME before resolution switching and aspect correction will work. Do this before enabling CRT SwitchRes so that RetroArch will run in your desktop resolution. Once you have rotated any games that may need it turn CRT SwitchRes on.
+
+## Socials
+
+The links below belong to our official channels. Links other than this may have been created by fans, independent members or followers. We seriously recommend using our original resources.
+
+- [Website](https://www.retroarch.com/)
+- [Blog](http://libretro.com/)
```
Comment (Margen67) [inline README.md]:
```suggestion
- [Blog](https://libretro.com/)
```

File: README.md [hunk 5]
```diff
@@ -256,3 +259,23 @@ These modelines are more accurate giving exact hz. However, some games may have
...
+- [Reddit](https://www.reddit.com/r/RetroArch/)
+- [YouTube](https://www.youtube.com/Libretro)
+- [Google Post](https://posts.google.com/share/55Nhs2jG)
+- [Steam](https://store.steampowered.com/app/1118310/RetroArch/)
+- [YouTube Topic](https://www.youtube.com/channel/UC5q007PYyQPgin0HHbzF0zQ)
+- [Patreon](https://www.patreon.com/libretro)
+- [BOUNTYSOURCE](https://www.bountysource.com/teams/libretro/issues)
+- [Discord](https://discord.gg/27Xxm2h)
+- [Teespring](https://teespring.com/stores/retroarch)
+- [Documentation](http://docs.libretro.com/)
```
Comment (Margen67) [inline README.md]:
```suggestion
- [Documentation](https://docs.libretro.com/)
```

**Comment by Margen67** [comment]
Commit history should be squashed.

**Comment by RobLoach** [comment]
Thanks all! Looks much nicer.

**Pull request merged**